frontend/mr-admin-flate/src/components/avtaler/OpprettAvtaleContainer.tsx

```diff
@@ -1,27 +1,32 @@
 import { zodResolver } from "@hookform/resolvers/zod";
 import { Button, Textarea, TextField } from "@navikt/ds-react";
 import classNames from "classnames";
-import { Avtale, AvtaleRequest, Avtaletype, Norg2Type } from "mulighetsrommet-api-client";
+import {
+  Avtale,
+  AvtaleRequest,
+  Avtaletype,
+  Norg2Type,
+} from "mulighetsrommet-api-client";
 import { Ansatt } from "mulighetsrommet-api-client/build/models/Ansatt";
 import { NavEnhet } from "mulighetsrommet-api-client/build/models/NavEnhet";
 import { Tiltakstype } from "mulighetsrommet-api-client/build/models/Tiltakstype";
-import { StatusModal } from "mulighetsrommet-veileder-flate/src/components/modal/delemodal/StatusModal";
 import { porten } from "mulighetsrommet-frontend-common/constants";
+import { StatusModal } from "mulighetsrommet-veileder-flate/src/components/modal/delemodal/StatusModal";
```

Comment:
Er dette kandidater for å trekke ut i et fellesbibliotek for frontend? Synes det virker litt snodig at `mr-admin-flate` har `mulighetsrommet-veileder-flate` som en depdency.

Reply:
Har sett at det gjelder et par andre tilfeller også, så evt. gjøres som en separat greie hvis det blir tungvindt?

Reply:
Det kan like gjerne være en feil-import. Jeg sjekker.